### DISCUSSION TITLE
收集你的「打呼聲資料」，調整枕頭高度！

### DISCUSSION THREAD

**Discussion by yi-1111** [1101-GS4719-C/Python-Community-GS4719-C#509] [科技新知]
韓國 Zerema 枕頭]結合 AI 深度學習來紀錄使用者的睡眠習慣，該枕頭配備 AI 驅動的打呼檢測技術，當枕頭感覺到使用者打呼時，就會自己提高或降低枕頭的高度，調整適當的呼吸形式，提供個性化的睡眠分析。

沒想到現在的枕頭也開始智能了，真期待那天的到來

https://buzzorange.com/techorange/2022/03/14/zerema-sleep/

**Comment by Sazmafuji** [comment]
我好像試試看，在用這個枕頭的人的旁邊，製造打呼聲，讓他枕頭一直上上下下，應該很好玩

**Comment by yucheng1028** [comment]
這樣可以解決室友打呼很吵的問題，這樣就不用解決室友了

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，這樣就可以知道自己的睡眠狀況。

**Comment by ghost** [comment]
看到這個就想到這個應該會用到有關音頻分析的東西，如何過濾雜訊之類的，真讓人挺好奇的

**Comment by sean1325418** [comment]
這樣終於可以不用晚上被吵醒去幫打呼的人調姿勢了。

**Comment by hapiwangy** [comment]
這類型的科技還需要配上精美的儀器製作，不然可能會因為調整的方式不當（例如器械的聲音太大聲）而造成其他的問題。期待他能解決大家的睡眠問題。

**Comment by ThomasMeng0515** [comment]
加上這些AI驅動的感測器可能會有噪音上的問題，而且說不定枕頭的重量也會提升，我個人還是保持觀望的態度

**Comment by yuhao112** [comment]
室友有這個問題，該叫他來看看了

**Comment by PoooYen** [comment]
感謝分享，不然我的室友睡著比醒著還吵，希望這對他有用

**Comment by xkllkx** [comment]
謝謝分享，這樣就不用被打呼吵醒了。

**Comment by Yuhsin000** [comment]
我覺得這個很實用，有時候過敏呼吸不順暢時都會擔心睡覺會打呼，有這個應該可以睡得很安心!

**Comment by etin2468** [comment]
這個東西也太讚，這樣我是不是就可以有個安穩的睡眠了

**Comment by SamTU0222** [comment]
謝謝分享，這個產品也太吸引人了吧

**Comment by lx3211** [comment]
感覺這個枕頭會硬硬的....這樣會不會不好睡呀

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望可以盡快使用到。

**Comment by henrywen547** [comment]
確實，枕頭高度相當影響睡眠品質，打呼也能判斷睡眠品質好壞，有這個枕頭真的對那些人幫助很大!

**Comment by jason79461385** [comment]
我倒是遇到比這個更嚴重的問題，室友因為太重還怎樣，整晚睡覺翻個身他的床組都發出超大的聲音⋯

**Comment by Lai27330** [comment]
感覺不只能夠解決打呼問題，還能改善睡眠品質，有時候真的會因為不同睡姿導致枕頭高度不適合而翻來翻去睡不著，有點吸引人。

**Comment by Rosancl** [comment]
這個對於有打呼狀況的人跟他的室友還滿方便的，但感覺故障的時候會很好笑。

**Comment by d26488399** [comment]
感谢分享，我爸也有打呼問題

**Comment by treeann** [comment]
打呼是我們全家都有的問題 謝謝分享ˋ

**Comment by 110409538** [comment]
我覺得這個枕頭很吸引我.因為我到現在也不清楚自己適合甚麼高度的枕頭.但是有了這項技術,就可以讓我有個適合自己的枕頭.

**Comment by bloodtrail0511** [comment]
超極有用!這樣就不用睡到一半再起來多加枕頭了

**Comment by stan890425** [comment]
室友的打呼聲我一率建議丟枕頭過去XD

**Comment by Jimmy911218** [comment]
這個ai的運用可能可以幫助道因睡眠問題困擾的人，感覺很實用

**Comment by HaoFish123** [comment]
感謝分享，感覺很實用，想實際體驗看看。

**Comment by minniewei** [comment]
我也想要體驗看看！！！

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by alexthebigpp** [comment]
跟另外一篇聽心跳聲改變音樂的科技感覺有異曲同工之妙
